app/views/observations/index.html.haml

```diff
@@ -147,9 +146,33 @@
                     = "{{ p.placeTypeLabel( ) }}"
                 %span{ "ng-show": "!nearbyPlaces.community || nearbyPlaces.community.length == 0" }
                   {{ shared.t("no_places_available") }}
-            %button.button.btn-default.reload{ "ng-click": "filterByBounds( )", "ng-show": "viewing('observations', 'map') && !hideRedoSearch" }
-              %span.glyphicon.glyphicon-repeat{ "aria-hidden": "true" }
-              {{ shared.t('redo_search_in_map') }}
+            %span.map-shape-searches
+              %div.shape-buttons
+                %button{
+                  "ng-click": "drawRegion( 'rectangle' )",
+                  "ng-show": "viewing('observations', 'map')",
+                  :class => "btn btn-default btn-rect {{ currentActiveDrawing != null ? 'is-active' : '' }}",
+                  "tooltip-enable" => "currentActiveDrawing == null",
+                  "uib-tooltip" => "{{ shared.t('draw_rectangular_range') }}"
+                }
+                  %img{ src: asset_path( "draw-rectangle-icon.png" ), "aria-hidden": "true" }
+                %button{
+                  "ng-click": "drawRegion( 'circle' )",
+                  "ng-show": "viewing('observations', 'map')",
+                  :class => "btn btn-default btn-circle {{ currentActiveDrawing != null ? 'is-active' : '' }}",
+                  "tooltip-enable" => "currentActiveDrawing == null",
+                  "uib-tooltip" => "{{ shared.t('draw_circular_range') }}"
+                }
+                  %img{ src: asset_path( "draw-circle-icon" ), "aria-hidden": "true" }
+              %button{
+                "ng-click": "filterByBounds( )",
+                "ng-show": "viewing('observations', 'map')",
+                :class => "btn btn-default btn-fit-to-map {{ disableFitToMap ? 'is-active' : '' }}",
+                "tooltip-enable" => "!disableFitToMap",
+                "uib-tooltip" => "{{ shared.t('search_current_map_range') }}",
+                "tooltip-class": "search-current-map"
+              }
+                {{ shared.t("fit_to_map") }}
```

Comment:
I'd like to stick with the current "Redo search in map". I know it's wide, but I don't think the meaning of "Fit to map" will be immediately obvious to most people at first.